main/test/ca/uwaterloo/flix/library/TestInt32.flix

```diff
@@ -594,7 +594,39 @@ namespace TestInt32 {
     
     @test
     def flipBit12(): Bool = Int32.flipBit(-1, 32) == -2
-    
+
+    /////////////////////////////////////////////////////////////////////////////
+    // log2                                                                    //
+    /////////////////////////////////////////////////////////////////////////////
+
+    @test
+    def log201(): Bool = Int32.log2(10) == 3
+
+    @test
+    def log202(): Bool = Int32.log2(10000) == 13
+
+    @test
+    def log203(): Bool = Int32.log2(8) == 3
```

Comment:
Can I ask you to add just a few more tests for both?

Reply:
Yes!